### PR TITLE
English homework "GRAMMA" has been donne

### PR DESCRIPTION
**Pull request by Svetlana101010**
Present Simple & Present Continuous.

### CODE REVIEW

**Review by Alex99905 — APPROVED** [review]

File: homework/english/svetlana_konduhovich/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+10. Is his wife having coffee at the moment?
+11. Let’s go for a walk. It isn't raining.
+12. Look! The boys are coming home.
+13. Does Maila like Chinese food?
+14. My grandmother doesn't work. She is on pension.
+15. What is your brother watching now?
+16. Where do you live?
+17. Do you play the piano well?
+18. Where is their brother swimming now?
+19. What are they eating at breakfast?
```
Comment (SvetaRomashko) [inline homework/english/svetlana_konduhovich/lecture1/task1.md:21]: Я сделала так:    What do they eat at breakfast?

File: homework/english/svetlana_konduhovich/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+22. Maila doesn't watch TV every morning.
+23. Listen! Blanca is singing in the bathroom.
+24. Bea sometimes reads comics.
+
+##Найдите ошибки и исправьте их.
+
+1. We are not going to school today.
+2. What do you do after school?
+3. At the moment Peter is working in Russia.
+4. Does he have a new car?
```
Comment (SvetaRomashko) [inline homework/english/svetlana_konduhovich/lecture1/task1.md:33]: У меня так:   Does he get a new car? or Has he got a new car?

**Review by SvetaRomashko — COMMENTED** [review]
See notes and answer for me, please.

**Review by anastasinastena — APPROVED** [review]

**Review by olgaz-skiba — APPROVED** [review]
I support comments

File: homework/english/svetlana_konduhovich/lecture1/task1.md (hunk 1 above)
Comment (Alex99905) [reply homework/english/svetlana_konduhovich/lecture1/task1.md:21]: Да, так правильно

**Pull request merged**